src/convertToObject.js

```diff
@@ -5,8 +5,19 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
```

Comment:
The function parameter is named `styles`, but the JSDoc comment refers to it as `sourceString`. Ensure consistency between the parameter name and the JSDoc comment.